Get Started/getstarted.html

```diff
@@ -41,19 +41,19 @@ <h3>Colors</h3>
               <span >Primary Color<br>var(--primary)</span>
             </div>
             <div class=" theme-color-block off-white"  >
```

Comment:
Could use BEM naming for easy understanding of class names.

Reply:
okay